src/main/java/player/Player.java

```diff
@@ -0,0 +1,44 @@
+package player;
+
+import map.Map;
+
+public class Player {
+
+    public class Position {
+        private int x;
+        private int y;
+
+        public int getPositionX() { return x; }
+        public int getPositionY() { return y; }
+        public boolean setPosition(int newX, int newY) {
```

Comment:
`public` access modifiers can be removed since methods will only be accessed from Player class and not from outside.